tree/20_cfe_basics/cfengine/cfe_internal.cf

```diff
@@ -39,100 +39,87 @@
 # the online docs
 #
 
-# For CFEngine Core: 3.5.0 to 3.5.x
-# Internal hub maintenance bundles
-
+# For CFEngine Core: 3.8.0 to 3.8.x
```

Comment:
i thought we were advertising compatibility for 3.7 ?

Reply:
We are, these comments are likely wrong (or it is a bug in the masterfiles). I will check with a 3.7 agent.